frontend/pages/Homepage/HostsSummary/HostsSummary.tsx

```diff
@@ -32,16 +41,39 @@ const HostsSummary = (): JSX.Element => {
 
   const labels = useSelector((state: IRootState) => state.entities.labels.data);
 
-  const macCount = labels[7] ? labels[7].count.toLocaleString("en-US") : "";
-  const windowsCount = labels[10]
-    ? labels[10].count.toLocaleString("en-US")
-    : "";
-  const linuxCount =
-    labels[8] && labels[9]
-      ? (labels[8].count + labels[9].count + labels[11].count).toLocaleString(
-          "en-US"
-        )
-      : "";
+  // Builtin labels from state populate os counts
+  const labelsWithIds: any = Object.keys(labels).map((id) => {
+    const index = Number(id);
+    return isNaN(index) ? null : labels[index];
+  });
+
+  const platformLabelsArray = labelsWithIds.filter((label: any) => {
+    return (
+      label &&
+      label.label_type === "builtin" &&
+      PLATFORM_STRINGS.includes(label.name)
+    );
+  });
+
+  const getCount = (platformTitles: string[]) => {
+    return reduce(
+      platformLabelsArray,
+      (acc: number, label: any) => {
+        return platformTitles.includes(label.name) && !isEmpty(label.count)
+          ? acc + label.count
+          : acc;
+      },
+      0
+    );
+  };
```

Comment:
Can this all simplify to essentially the following?
```suggestion
  const getCount = (platformTitles: string[]) => {
    return reduce(
      Object.values(labels),
      (acc, label) => {
        return label.label_type === "builtin" && platformTitles.includes(label.name) && !isEmpty(label.count)
          ? acc + label.count
          : acc;
      },
      0
    );
  };
  };
```